modules/ROOT/pages/upgrade.adoc

```diff
@@ -4,88 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 
 == Before You Begin
```

Comment:
Maybe remove this header?

Reply:
I added a statement that resembles the install one - please take a look.